cmd/podman/containers/run.go

```diff
@@ -70,6 +70,10 @@ func runFlags(cmd *cobra.Command) {
 	flags.UintVar(&runOpts.PreserveFDs, "preserve-fds", 0, "Pass a number of additional file descriptors into the container")
 	_ = cmd.RegisterFlagCompletionFunc(preserveFdsFlagName, completion.AutocompleteNone)
 
+	preserveFdFlagName := "preserve-fd"
+	flags.UintSliceVar(&runOpts.PreserveFD, "preserve-fd", nil, "Pass a list of additional file descriptors into the container")
```

Comment:
I think you meant to use `preserveFdFlagName` here as the second arg?

Reply:
I've copied the same pattern that is used by `preserveFdsFlagName`.  I'll fix both